assignments/run_callback/run_callback.js

```diff
@@ -1,5 +1,14 @@
+function cb(output){
+    // console.log(output);
+    return output;
```

Comment:
why u returning the output in third line .These should  align in proper manner,

Reply:
Updated the code
https://github.com/ashokdv/JS-Assignments/pull/4/commits/f272fc694c203355854fef829e33f43828dcc15b